File: legacy/check.js
const fs = require('fs');
const txt = fs.readFileSync('C:\\Users\\javie\\.gemini\\antigravity\\scratch\\lozanor-app\\index.html', 'utf8');
const idx = txt.indexOf('yyyy}-${pad2');
console.log('Char code is: ' + txt.charCodeAt(idx - 3));
console.log('Char is: ' + txt.charAt(idx - 3));

File: legacy/check.cs
using System;
using System.IO;

public class Program {
    public static void Main() {
        string txt = File.ReadAllText(@"C:\Users\javie\.gemini\antigravity\scratch\lozanor-app\index.html");
        int idx = txt.IndexOf("yyyy}-${pad2");
        if(idx == -1) { Console.WriteLine("Not found"); return; }
        char c = txt[idx - 3];
        Console.WriteLine("Char code is: " + (int)c);
        Console.WriteLine("Char is: " + c);
    }
}
